### ISSUE TITLE
[FEAT] Enum Resolvers 

### ISSUE THREAD

**Issue by Teddy-Wilson** [eddeee888/graphql-code-generator-plugins#265]
**Is your feature request related to a problem? Please describe.**
Unless I've missed it theres currently no way to generate resolvers for GraphQL enum types.

**Describe the solution you'd like**
1. add enums to the `resolverGeneration` object
2. generate enum resolvers files.

**Describe alternatives you've considered**
None. But open to other alternatives if they exist.

**Additional context**
N/A

**Comment by eddeee888** [comment]
Hi @Teddy-Wilson ,

This feature is implemented in this alpha version `pr293-run533-1`.

Could you please install it and try it out please? 🙏

```ts
yarn add -DE @eddeee888/gcg-typescript-resolver-files@pr293-run533-1
```

**Comment by eddeee888** [comment]
This is now implemented in [v0.10](https://github.com/eddeee888/graphql-code-generator-plugins/releases/tag/%40eddeee888%2Fgcg-typescript-resolver-files%400.10.0)
